sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -5,6 +5,12 @@
 #include "azure/core/azure.hpp"
 #include "azure/core/http/http.hpp"
 
+#ifdef POSIX
```

Comment:
https://stackoverflow.com/questions/11350878/how-can-i-determine-if-the-operating-system-is-posix-in-c ?

Reply:
Is that only for GCC? What about other compilers?